users/views.py

```diff
@@ -53,13 +54,21 @@ def post(self, request):
         algorithm     = ALGORITHM
         data          = json.loads(request.body)
 
+        @unique
+        class SignUpType(Enum):
+            FLIX   = auto()
+            KAKAO  = auto()
+            GOOGLE = auto()
```

Comment:
@unique와 auto()를 사용하신 이유가 있을까요?

```suggestion
        @unique
        class SignUpType(Enum):
            FLIX   = auto()
            KAKAO  = auto()
            GOOGLE = auto()
```

Reply:
그리고 해당 클래스는 models.py로 옮겨 주세요~

Reply:
1, 2, 3 등 으로 임의의 수를 부여해서 보이게 하는 방법을 처음에 봤을 때, "왜 FLIX 는 1이고, KAKAO 는 2인거지?" 하는 생각이 들었습니다. 구분자를 부여하는거라면 auto() 자동생성 기능으로 혼란을 막고싶었는데... 생각해보면 굳이... 라는 느낌도 있네요. 

뷰 파일에 Enum 클래스를 붙여둔 이유는,
장고 문서에서 Enum 을 모델파일에 사용한 것을 보고 그대로 적용할까 했는데 사용하는 곳이 View 중에서도 Sign-In 부분밖에 없다고 생각했습니다. 그럴 바엔 뷰의 Sign-In 부분에 서브 클래스로 붙여두는 게 직관적일거라 생각했습니다. 다시 돌아보니 다른 방식의 하드코딩이 되어버린 것 같네요.

고쳐서 다시 PR 올리겠습니다!